packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/ExtractionService.java

```diff
@@ -73,32 +73,40 @@ public class ExtractionService {
 	// time the progress takes to reach each subsequent half
 	final Double HALFTIME_SECONDS = 2.0;
 
+	// @Value("${terarium.extractionService.poolSize:10}")
```

Comment:
This property is still in application.properties - you should be able to use this and get rid of the hard coded pool size.

Reply:
I updated it. I think the changes you made in my previous PR are lost so I added it back.